### ISSUE TITLE
Speech recognition

### ISSUE THREAD

**Issue by lazna** [carlos-montiers/enhancedbatch#56]
After the (already implemented) '@say' got another crazy idea: Is it possible to console scripts react according to voice? 

Have large, menu driven, shell script reacting on specific keypress utilizing choice program. Have idea of  something like @getkb but it wait for recognized voice output too and send it into the stdout, just what come first: keypress or voice.

It could be handy so technicians could have both hands free (for example holding heavy things) but see laptops screen and can say trivial tasks to the headset, like 'ping some_ip'...

Is it possible at all?

https://outnull.wordpress.com/2016/11/14/powershell-speech-recognition/

**Comment by adoxa** [comment]
You're asking me if it's possible, when you link to a PowerShell script that does it! Use PowerShell!

**Comment by carlos-montiers** [comment]
I agree with @adoxa. Have a speech tool. For example, the old Japanese computer PC-6001 mk2 using N60m/N66-BASIC has "TALK" command.
But have a tool for recognizing the voice, is a feature too much specialized for be included in EB.

Mmm what about rename SAY to TALK?

**Comment by adoxa** [comment]
Talk is what you do, say is what you speak.

**Comment by lazna** [comment]
> After the (already implemented) '@say' got another crazy idea: Is it possible to console scripts react according to voice?
> 
> Have large, menu driven, shell script reacting on specific keypress utilizing choice program. Have idea of something like @getkb but it wait for recognized voice output too and send it into the stdout, just what come first: keypress or voice.
> 
> It could be handy so technicians could have both hands free (for example holding heavy things) but see laptops screen and can say trivial tasks to the headset, like 'ping some_ip'...
> 
> Is it possible at all?
> 
> https://outnull.wordpress.com/2016/11/14/powershell-speech-recognition/



> You're asking me if it's possible, when you link to a PowerShell script that does it! Use PowerShell!

1. System prior W10 may not have PS installed
2. In @say command, PS it 10 times slower than your EB solution, expect similar issue on speech recognition.
